File: Java/1925. Count Square Sum Triples.java
class Solution {
    public int countTriples(int n) {
        int ans = 0;
        double temp;

        for(int i = 1; i < n; i++){
            for(int j = 1; j < n; j++){
                temp = Math.sqrt(i * i + j * j);
                if(temp <= n && temp == (int)temp) ans++;
            }
        }

        return ans;
    }
}

File: Python/1925. Count Square Sum Triples.py
class Solution:
    def countTriples(self, n: int) -> int:
        res = 0

        for i in range(1, n):
            for j in range(1, n):
                temp = (i * i + j * j) ** 0.5
                if temp <= n and temp.is_integer(): res += 1
        
        return res
